api/hds.proto

```diff
@@ -52,6 +52,10 @@ service HealthDiscoveryService {
       returns (stream HealthCheckSpecifier) {
   }
 
+  rpc BidiStreamHealthCheck(stream HealthCheckRequest)
```

Comment:
This is already a bidi API and I'm not sure if it benefits from versioning. It's really an ongoing stream of management server initiated requests for health checks and Envoy responses. There's no persistence really (or impact on Envoy's proxying).